omod/src/main/webapp/resources/htmlforms/071-IntegratedAntenatalRegister.xml

```diff
@@ -507,6 +509,12 @@
 	            	valid = false;
 	            }
 
+                if (encounterDate &gt;= returnDate) {
+                getField('returndate.error').html('The next appointment date must be after the date of this visit ' +
+                encounterDate).show();
```

Comment:
@paleu256 format the encounterDate in the error message now its showing a very long format